NewForce-at-Mountwest/bangazon-workforce-gnocci
# Add Employee Ticket, Employee Details Ticket

Ticket 2
https://trello.com/c/rbysDIzr/2-hr-should-be-able-to-add-an-employee

Ticket 3
https://trello.com/c/BSyUkXQs/3-hr-should-be-able-to-view-an-employees-details

## Steps to Test

Before you start: Run the SQL SEED DATA found in the .md file.
1- Press Ctrl+f5
2- Click on the Employees tab
3- You should see all Employees listed with their first name, last name, and department.
4- Click 'Create New'
5- Fill out the form then click create
6- The new employee should show at the bottom of the employee tab
7- Click on 'Details' next to an employee
8- You should see a page with the employees details

# Checklist:
- [x] I learned something
